cdap-docs/cdap-apps/source/hydrator/plugins/third-party.rst

```diff
@@ -26,16 +26,16 @@ A sample JDBC Driver Plugin configuration:
       "parents": [ "cdap-etl-batch[|version|,\ |version|]" ],
       "config": [
         {
-          "type" : "JDBC",
-          "name" : "MySQL JDBC",
-          "description" : "Plugin for MySQL JDBC driver",
+          "name" : "mysql",
+          "type" : "jdbc",
           "className" : "com.mysql.jdbc.Driver"
+          "description" : "Plugin for MySQL JDBC driver",
```

Comment:
if we're moving description down, need to remove the trailing comma and add it to the end of the previous line to ensure its valid json